### DISCUSSION TITLE
1.30.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#16455] [General]
# Notes
- For CC2652/CC1352 users: a new firmware is available which should improve preformance with larger networks ([`20221226`](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin)).

# Features
- [#5276](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5276) Add `state_action` option, this allows switches to send state changes also as an `action` (@Mr-Groch)

# Improvements
- [#5241](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5241) Support timer for Sinopé DM2500 and SW2500 (@onixx)
- [#5243](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5243) Expose `led_in_dark` setting for Legrand 067776 (@AntorFr)
- [#5276](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5267) [#5245](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5245) [#5312](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5312) [#15669](https://github.com/Koenkk/zigbee2mqtt/issues/15699) [#5290](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5290) [#5311](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5311) Enable Hue effects for more Philips lights (@marblecoma, @merlinschumacher, @deviantintegral, @Koenkk, @Sabesto, @kris-anderson)
- [#5242](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5242) Support more features for Xiaomi WS-USC04 (@visualage)
- #15872 Support Hue gradient feature for more bulbs (@zegl)
- [#5253](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5253) [#16147](https://github.com/Koenkk/zigbee2mqtt/issues/16147) [#5315](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5315) [#6180](https://github.com/Koenkk/zigbee2mqtt/issues/16180) Support `power_on_behavior` for more switches (@Koenkk, @slippern1) 
- [#5264](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5264) [#5277](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5277) Expose apparent power for some Legrand devices (@Mephistofeles, @pierrepinon)
- [#5265](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5265) Improve Inoveilli VZM31-SN energy report interval (@InovelliUSA)
- [#5272](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5272) Set color temperature range for Philips 8718696598283, 929001953301 and 9290019534 (@zegl)
- [#5283](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5283) Add OTA support for Bosch BTH-RA (@compujuckel)
- [#5289](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5289) Support `led_in_dark` and `identify` for BTicino K4003C/L4003C/N4003C/NT4003C, L4411C/N4411C/NT4411C, K4027C/L4027C/N4027C/NT4027C (@knovoselic)
- [#16017](https://github.com/Koenkk/zigbee2mqtt/pull/16017) Expose cover moving state to Home Assistant (@cmiguelcabral)
- [#5296](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5296) Support additional features for Schneider Electric EER50000 (@pklokke)
- [#5310](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5310) Read `hw_version` of Datek HSE2905E (@TheStigh)
- [#16217](https://github.com/Koenkk/zigbee2mqtt/issues/16217) Support `energy` for Sengled Z01-A19NAE26 (@Koenkk) 
- [#5355](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5355) Support `min_brightness`, `max_brightness` and `countdown` for TuYa TS0601_dimmer_2  (@pyrexfm)
- [#13711](https://github.com/Koenkk/zigbee2mqtt/issues/13711) Basic region support for Xiaomi RTCZCGQ11LM (Aqara FP1) (@mdziekon, @patmalcolm91,  @Otnow)
- [#240](https://github.com/Koenkk/zigbee-OTA/pull/240) Support OTA for Datek HLU2909K (@cskogstrand)
- [#16300](https://github.com/Koenkk/zigbee2mqtt/pull/16300) Sane defaults for remote syslog (@mtdcr)
- [#5299](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5299) Improvements for Busch-Jaeger 6735/6736/6737 (@klada)
- [#5384](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5384) Expose `power`, `current`, `voltage` and `energy` for Samotech SM309-S (@alexjfisher)
- [#5376](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5376) Support more features for Sinope thermostats and fix reporting of energy for Sinope plugs (@MattL0)
- [#5414](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5414) Support more features for Yale locks (@joshuaspence)
- [#5398](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5398) Support `throw` `action` for Xiaomi MFKZQ01LM (@JJPro)
- [#5401](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5401) Expose `battery` and fix power source of Third Reality 3RSS009Z (@grelca) 
- [#16371](https://github.com/Koenkk/zigbee2mqtt/issues/16371) Always respond to OTA request (@Koenkk) 
- [#650](https://github.com/Koenkk/zigbee-herdsman/pull/650) EZSP adapter reliability improvements (@kirovilya)

# Fixes
- [#15856](https://github.com/Koenkk/zigbee2mqtt/issues/15856) Fix many database writes causing instability (@Koenkk) 
- [#15849](https://github.com/Koenkk/zigbee2mqtt/issues/15849) Detect `_TZ3000_mlswgkc3` as ZIGBEE-B09-UK (@Koenkk) 
- [#15851](https://github.com/Koenkk/zigbee2mqtt/discussions/15851) Fix 3315-Geu vendor (Iris -> Centralite) (@Koenkk)
- [#15554](https://github.com/Koenkk/zigbee2mqtt/issues/15554) [#5295](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5295) [#5306](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5306) [#16153](https://github.com/Koenkk/zigbee2mqtt/issues/16153) Disable unsupported `power_on_behavior` for Lidl lights, Sonoff ZBMINI, SONOFF S31ZB, Miboxer FUT036Z (@Koenkk, @stickpin, @digiblur)
- [#5260](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5260) Fix GIDEALED ZC05M maximum color temperature (@Cyberbeni) 
- [#5270](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5270) Fix state problems for Eurotronic SPZB0001 (@kezmdor) 
- [#5271](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5271) Disable power on behavior for some Dawon DNS devices @nukusinji
- [#15946](https://github.com/Koenkk/zigbee2mqtt/issues/15946) Rename TuYa TS0601_soil humidity to soil_moisture (@Koenkk) 
- [#15957](https://github.com/Koenkk/zigbee2mqtt/issues/15957) Detect `_TZ3000_gjrubzje` as TuYa TS0001_switch_module (@Koenkk) 
- [#5281](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5281) Fix Ubisys C4 ations when in non legacy mode (@sjorge) 
- [#15958](https://github.com/Koenkk/zigbee2mqtt/issues/15958) Fix Home Assistant `Entity sensor.xxx (<class 'homeassistant.components.mqtt.sensor.MqttSensor'>) is using native unit of measurement '' which is no[#15958](https://github.com/Koenkk/zigbee2mqtt/issues/15958) t a valid unit for the device class ('illuminance') ` warning (@Koenkk) 
- [#5246](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5246) Fix OTA update available check when using force (@rccoleman)
- [#5286](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5286) Fix Legrand 412015 configure failing (@sjorge)
- [#5287](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5287) Fix electrical measurements not working for TuYa TS011F_plug with appVersion 100 (@egor-n) 
- [#15979](https://github.com/Koenkk/zigbee2mqtt/issues/15979) Fix TERNCY-DC01 detected as SR-ZG9010A  (@DubhAd)
- [#5288](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5288) Disable unsupported `power_on_behavior` for ORAM AB3257001NJ and AC10691 (@stickpin)
- [#4859](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4859) Fix warnings generated by TuYa MIR-HE200-TY (@Koenkk) 
- [#5294](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5294) Fix configure of OWON PIR313-E (@kirovilya) 
- [#5305](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5305) Fix Lidl HG08164 actions (@ghhassta) 
- [#5319](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5319) Fix time units of Moes BRT-100-TRV exposes (@tefracky) 
- [#16010](https://github.com/Koenkk/zigbee2mqtt/issues/16010) Fix `battery` and `voltage` not updating of TuYa IH012-RT01 (@Koenkk)
- [#5323](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5323) Fix HKGK BAC-002-ALZB `local_temperature` factor 10 too low (@VoltickVL)  
- [#5326](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5326) Fix no `energy` reported for Sinopé SP2610ZB and SP2600ZB (@MattL0) 
- [#15472](https://github.com/Koenkk/zigbee2mqtt/issues/15472) Fix Custom devices (DiY) ti.router `transmit_power` off by one (@jerrm) 
- [#5327](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5327) Fix `fan_mode` and `fan_state` value incorrect (@davidjb)
- [#16132](https://github.com/Koenkk/zigbee2mqtt/issues/16132) Fix Eglo Fueva-Z mis-identified (@Koenkk)  
- [#5336](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5336) Remove unsupported color from Lidl 14156408L (@MichelFR) 
- [#5316](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5316) Fix missing OTA cluster of Aqara SRTS-A01 (@protyposis) 
- [#5345](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5345) Fix power source of Nue / 3A NUE-ZBFLB (@fredskis) 
- [#16207](https://github.com/Koenkk/zigbee2mqtt/issues/16207) Fix `Failed to call 'Frontend' 'stop' (TypeError: Cannot read properties of null (reading 'clients')` error (@Koenkk)
- [#16090](https://github.com/Koenkk/zigbee2mqtt/issues/16090) Fix cover state of Legrand 067776 (@Koenkk) 
- [#15902](https://github.com/Koenkk/zigbee2mqtt/issues/15902) Fix TuYa TS110E_1gang_1 not turning off with physical switch (@Koenkk)
- [#13600](https://github.com/Koenkk/zigbee2mqtt/issues/13600) Fix SONOFF SNZB-02D configure (@Koenk)
- [#16216](https://github.com/Koenkk/zigbee2mqtt/issues/16216) Detect `_TZ3000_4o16jdca` as TuYa TS0003_switch_module (@Koenkk) 
- [#5352](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5352) Fix calibration limits of Bosch BTH-RA (@Master92) 
- [#16239](https://github.com/Koenkk/zigbee2mqtt/issues/16239) Fix not able to trigger feed through Home Assistant for Xiaomi ZNCWWSQ01LM (@kimmeld) 
- [#5364](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5364) Fix Xiaomi ZNCWWSQ01LM time (@kirovilya) 
- [#5365](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5365) Fix DIYRuZ DIYRuZ_rspm missing actions (@o0shojo0o)
- [#5363](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5363) Fix reported battery and firmware version for Aqara SRTS-A01 (@protyposis) 
- [#5372](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5372) Fix Moes BHT-002-GCLZB `current_heating_setpoint` 10 times lower than expected for `_TZE200_5toc8efa` (@VoltickVL) 
- [#5375](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5375) Remove redundant `temperature` from Schneider Electric EER51000 (@pklokke)  
- [#5371](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5371) Remove unsupported `off` `system_mode` and fix step size for Moes BRT-100-TRV (@gitaen) 
- [#16201](https://github.com/Koenkk/zigbee2mqtt/issues/16201) Fix Docker image incompatible with e.g. Watchtower (@rramthun) 
- [#5385](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5385) Fix Danfoss 014G2461 `window_open_internal` expose (@Nezz) 
- Sanatize `manufacturerName`, https://github.com/home-assistant/core/issues/85691 (@Koenkk) 
- [#5395](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5395) Remove unsupported `auto` `system_mode` fro SMaBiT AV2010/32 (@sjorge) 
- [#16346](https://github.com/Koenkk/zigbee2mqtt/issues/16346) Fix min/max `local_temperature_calibration` range for various thermostats (@Koenkk) 
- [#16348](https://github.com/Koenkk/zigbee2mqtt/issues/16348) Fix Paulmann 501.34 dimming commands not working (@IngoK1) 
- [#5399](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5399) Remove unsupported `cool` `running_state` from SMaBiT AV2010/32 (@sjorge) 
- [#14628](https://github.com/Koenkk/zigbee2mqtt/issues/14628) Fix dimming of TuYa `_TZ3210_pagajpog` and `_TZ3210_4ubylghk` not working (@Koenkk) 
- [#5388](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5388) Fix TuYa ZS06 errors (@Legogris) 
- [#5405](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5405) Fix backlight settings of Moes ZS-EUB_1gang (@mcuelenaere) 
- [#16412](https://github.com/Koenkk/zigbee2mqtt/issues/16412) Fix IKEA E1524/E1810 battery percentage multiplied by two (@Koenkk) 
- [#5409](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5409) Remove unsupported `battery_low` from Sinope RM3500ZB and improve reporting configuration (@embak) 
- [#5402](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5402) Fix KAMI N20 `occupancy` (@Tediore) 

# New supported devices
This release adds support for 74 devices:
- `067766` Legrand Centralized ventilation switch (@AntorFr)
- `2189/1-xx` Bankamp Grazia ceiling light (@ahemwe)
- `4076130P6` Philips Hue white ambiance suspension Cher with bluetooth 3000lm (@zegl)
- `42-032` LightSolutions LED driver CCT 12V - 30W - CCT (@lbach)
- `43109` Enbrighten Zigbee in-wall smart switch (@jacobefaust)
- `4512749` Namron Thermostat outlet socket (@slippern1)
- `50208702` Sunricher LED dimmable driver (@Koenkk)
- `511.324` Iluminize Zigbee handheld remote CCT 4 channels (@vojtechBohman)
- `578526` Philips Hue white ambiance extra bright high lumen dimmable LED smart retrofit recessed 6" downlight (@syncnj)
- `81863` AduroSmart Eria color LED strip (@m00n3n)
- `8719514279155` Philips Hue white G125 B22 LED bulb filament giant globe (@zegl)
- `8719514301443` Philips Hue White Ambiance B22 filament screw globe (@Koenkk)
- `915005987701` Philips Hue Gradient Signe floor lamp (wood) (@moshemalawach)
- `915005987901` Philips Hue Gradient light tube compact (@m4k5ym)
- `915005988101` Philips Hue Gradient light tube large (@m4k5ym)
- `9290024782` Philips Hue G125 B22 White Ambiance filament bulb (@sdriessens)
- `9290029808` Philips Hue white filament Edison ST23 E26 LED warm-to-cool (@bockbilbo)
- `929003047001` Philips Hue White ambiance Milliskin (round) (@adamlc)
- `929003047601` Philips Hue White and Color Ambiance GU10 (Centura) (@xPapla)
- `9290030517` Philips Hue White Ambiance E27 ST64 filament bulb (@Koenkk)
- `929003054501` Philips Hue white ambiance Fair ceiling with Bluetooth (white) (@zegl)
- `929003054701` Philips Hue white ambiance Fair ceiling with Bluetooth (black) (@zegl)
- `929003074801` Philips Hue White and Color Ambiance GU10 (Xamento) (@zegl)
- `929003115801` Philips Hue Perifo ceiling light, 1 pendant (white) (@erikwout)
- `929003116201` Philips Hue Perifo linear light bar (@erikwout)
- `929003117101` Philips Hue Perifo ceiling light, 3 pendant (black) (@zegl)
- `929003117201` Philips Hue Perifo ceiling light, 3 pendant (white) (@zegl)
- `929003117301` Philips Hue Perifo ceiling light, 4 spotlights (black) (@zegl)
- `929003117401` Philips Hue Perifo ceiling light, 4 spotlights (white) (@zegl)
- `929003117701` Philips Hue Perifo Wall Light, 3 spotlights (black) (@zegl)
- `929003117801` Philips Hue Perifo Wall Light, 3 spotlights (white) (@zegl)
- `9290031451` Philips Hue white ambiance filament E12 with Bluetooth (@jwekwert)
- `929003516101` Philips Hue Gradient Signe floor lamp (wood) (@rabb13)
- `929003531702` Philips Hue Bluetooth white & color ambiance ceiling lamp Infuse medium (@zegl)
- `AL8RGB13W-AP` Alchemy Downlight RGBW (@Koenkk)
- `BSEN-C2` Bosch Door/window contact II (@g-jaeger)
- `BSP-EZ2` Bosch Plug compact FR (@Koenkk)
- `BSP-FZ2` Bosch Plug compact EU (@Master92)
- `CCT591011_AS` Schneider Electric Wiser window/door sensor (@onoki)
- `CTP-R01` Xiaomi Aqara magic cube T1 Pro (@JJPro)
- `DCH-B112` D-Link Wireless smart door window sensor with vibration (@simongoudie)
- `DZ8` Hilux Spot 7W (@ChristianHC)
- `EB-E14-P45-RGBW` EssentielB Smart LED bulb (@Gmax76)
- `G2015` IKEA PILSKOTT LED floor lamp (@TimoRemon)
- `HA-ZM12/24-mw2` Halemeier MultiWhite 1-channel smart receiver 12V (@FeKoerner)
- `HAL800` Schwaiger LED candle bulb E14 470 lumen, dimmable, color, white 1800-6500K (@TheNetStriker)
- `HS2SS-E_V03` HEIMAN Smart doorbell button (@hugokernel)
- `HS3HT` HEIMAN Temperature & humidity sensor with display (@threnard)
- `MEG5126-0300` Schneider Electric Merten MEG5165 PlusLink shutter insert with Merten Wiser System M push button (2fold) (@pfandfrei)
- `ROB_200-061-0` ROBB 50W Zigbee CCT LED driver (constant current) (@Koenkk)
- `SA100` Cleverio Smart siren (@blackcult)
- `SB100` Cleverio Wireless switch with 1 button (@blackcult)
- `SEHAZB-DR-SWITCH-2` SolarEdge Smart energy switch (@rutgerellen)
- `SIN-4-1-20_PRO` NodOn Single LED relay (@Char-r)
- `SIN-4-2-20_PRO` NodOn Double LED relay (@Char-r)
- `SM324` Samotech 220V Zigbee CCT LED dimmer (@iqbalibrahim1992)
- `SMA03P` Mercator Ikuü temperature & humidity sensor (@kiwizznz)
- `SMA04P` Mercator Ikuü battery contact sensor (@kiwizznz)
- `SMCL01-ZB` Mercator Ikuü Ikon ceiling light CCT (@kiwizznz)
- `SMD4109W-RGB-ZB` Mercator Ikuü Walter downlight RGB + CCT (@kiwizznz)
- `SPP02G` Mercator Ikuü double indoors power point (@kiwizznz)
- `SSW03G` Mercator Ikuü triple switch (@kiwizznz)
- `T2030` IKEA PILSKOTT LED pendant lamp (@GsakuL)
- `TH1124ZB-G2` Sinopé Zigbee line volt thermostat (@spyrolabs)
- `TS0002_power` TuYa 2 gang switch with power monitoring (@Koenkk)
- `TS0601_cover_2` TuYa Curtain motor fixed speed (@hotchkj)
- `TS110E_2gang_2` TuYa 2 channel dimmer (@Koenkk)
- `W599001` Schneider Electric Wiser smoke alarm (@dennisgranasen)
- `WNAL10/WNRL10` Legrand Smart switch with Netatmo (@brantmacga)
- `WNAL50/WNRL50` Legrand Smart dimmer switch with Netatmo (@brantmacga)
- `WNAL63` Legrand Remote dimmer switch (@brantmacga)
- `WS01` eWeLink Rainfall sensor (@snow6326)
- `ZB-CCT_Filament` Ajax Online Zigbee LED filament light dimmable E27, edison ST64, flame 2200K (@cooper645)
- `ZBCurtain` SONOFF Zigbee smart curtain motor (@Adam-Ant)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.30.0'>1.30.0</a>.</em>

**Comment by ahd71** [comment]
I've been running the new firmware for a few days now in a 200+ nodes network (zzh stick as controller) and it works excellent. It indeed addressed a few issues I had where the network took some time to deliver some messages intermittently (e.g some motion sensors and ikea blinds).

Excellent work. Thank you very much!

**Reply by cloudbr34k84** [reply]
When you mean 200 nodes? Do you mean 200 devices??

**Reply by ahd71** [reply]
Yes!
![Screenshot_20230202_080509_Chrome](https://user-images.githubusercontent.com/1518011/216255226-c5e2ba64-5667-49ef-a7ee-20f823dcf93a.jpg)

**Comment by MirFei** [comment]
Not very stable, service is easy to be down

**Comment by MirFei** [comment]
Assertion failed: Command (setValue) returned unexpected state: [object Object]
Assertion failed: Command (setValue) returned unexpected state: [object Object]
Zigbee2MQTT:error 2023-02-02 09:30:10: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-02-02 09:30:10: Failed to start zigbee
Zigbee2MQTT:error 2023-02-02 09:30:10: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-02-02 09:30:10: Exiting...
Zigbee2MQTT:error 2023-02-02 09:30:10: Error: Failure send getKey:{"type":"Buffer","data":[30,0,1,106,0,1]}
    at E:\houtai\zigbee2mqtt-1.28.0\node_modules\zigbee-herdsman\src\adapter\ezsp\driver\ezsp.ts:507:23
    at Queue.executeNext (E:\houtai\zigbee2mqtt-1.28.0\node_modules\zigbee-herdsman\src\utils\queue.ts:32:32)

**Reply by MirFei** [reply]
Error: Failure send getEui64:{"type":"Buffer","data":[29,0,1,38,0]}
    at E:\houtai\zigbee2mqtt-1.28.0\node_modules\zigbee-herdsman\src\adapter\ezsp\driver\ezsp.ts:507:23

**Reply by MirFei** [reply]
Received message from unsupported device with Zigbee model 'TS011F' and manufacturer name '_TZ3000_0zfrhq4i'

**Reply by ahd71** [reply]
Your log say v1.28.0 for some reason (in the discussion thread about 1.30 0)

**Comment by tamer82salah** [comment]
hi dear koenkk many thanks for the great work , can you Disable unsupported  power_on_behavior for the following please :

- osram model AC03641 Classic A60 W clear - LIGHTIFY
- Lonsonho model ZB-RGBCW eWeLight
 because these devices not supported on poweron behaviour and causing errors 

Best Regards

**Comment by dickthespoon** [comment]
I'm finding this has broken the Zigbee2MQTT web page in the HA addon, just gives blank page now, multiple restarts of the addon and reboots of the host device all give the same effect.

**Comment by antunespa** [comment]
After updating zigbee2mqtt my shutter modules (QS-Zigbee-C03) started to work incorrectly, when they are going up or down they don't report the percentage and when they finish opening or closing the home assistant keeps saying opening or closing, it seems that zigbee2mqtt dont report the state correctly to HA. After restoring the backup everything was fine again.
